application/src/command.rs

```diff
@@ -37,15 +36,12 @@ impl<R> Execute<R> for Command
 where
     R: TodoListRepository + TodoListStore + Send + Sync,
 {
-    type Error = AnyError;
-    type Output = ();
-
     async fn execute(&self, runtime: &R) -> AnyResult<()> {
-        let message = self.try_into()?;
+        let message: TodoListMessage = self.try_into()?;
```

Comment:
**Suggestion:** Consider using `?` operator directly on `self.try_into()` to propagate the error up and remove the need for `Ok`. [best practice]
```suggestion
        let message: TodoListMessage = self.try_into().map_err(|e| e.to_string())?;
```